### PR TITLE
test cases for components in small talk

### PR DESCRIPTION
**Pull request by nguyenj21**
### Issue Number
#182 

### Description
Added test cases for the following files **censorMess.js , data.js, footer.js , logo,js , sushi.js**
100% coverage

### Testing

Test cases for components section. Still need to finish header.js & layout.js test cases

### Possible Errors
Still need to finish header.js & layout.js test cases, other than that no error

image

### CODE REVIEW

**Review by junxjr — APPROVED** [review]
Everything looks good. Great job on making the test cases. There are still some more tests left, so we should all get the test cases done to meet the requirement. I see you added the Ai generate code comment, I think it really saves a lot of time to set up the test. I should use it more when making test cases for other pages.

**Review by salazw1 — APPROVED** [review]
Nice job on increasing our overall test coverage. I appreciate the transparency on your use of AI generation.

File: small-talk/components/censorMess.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import censor from './censorMess.js';
+//ai-gen start (ChatGPT-4, 1)
+// Mock the swearWords module
+jest.mock('./swearWords.js', () => {
+    return ['badword', 'anotherbadword'];
+})
+
+describe('censor function', () => {
+    it('should replace swear words with asterisks', () => {
+        const input = 'This is a badword and anotherbadword in a sentence.';
```
Comment (aiharc1) [inline small-talk/components/censorMess.test.js:10]: Im curious - how does this work if 'badword' and 'anotherbadword' is not in the dictionary of swearWords?

**Review by aiharc1 — APPROVED** [review]
Nice work here Johnson and using AI to our advantage (and documenting it). Way to get the coverage up!

**Review by 39otsu — APPROVED** [review]
I'm always learning from your test cases! I wonder how you would target layout.js when you've targetted the components that make up the layout component

**Review by AlvaroH2001 — APPROVED** [review]
Great job on adding these test cases, created and very needed.

**Pull request merged**